.travis.yml

```diff
@@ -102,4 +102,4 @@ deploy:
     # all_branches is required to use tags
     all_branches: true
     # Only publish the build marked with "DEPLOY_TO_FORGE"
-    condition: "$DEPLOY_TO_FORGE = yes"
+    condition: "$DEPLOY_TO_FORGE = yes"
```

Comment:
in general we want trailing newlines (documented at https://voxpupuli.org/docs/#reviewing-a-module-pr )

Reply:
Is there a chance to trigger a travis ci run, without changing the PR? I just removed the line to trigger a rebuild.

I move the dtailed discussion to a new issue

Reply:
you can always rerun travis by closing and reopening a PR

Reply:
Thanks